pkg/kobject/kobject.go

```diff
@@ -92,6 +92,7 @@ type ServiceConfig struct {
 	MemLimit        yaml.MemStringorInt `compose:"mem_limit" bundle:""`
 	TmpFs           []string            `compose:"tmpfs" bundle:""`
 	Dockerfile      string              `compose:"dockerfile" bundle:""`
+	Replicas        int                 `compose:"replicas" bundle:""`
```

Comment:
We want to replicate how we do it similarly with `kobject` in libcompose / docker/cli. 

I ask that you create a separate struct, similar to https://github.com/docker/cli/blob/master/cli/compose/types/types.go#L159 in order to implement the `Deploy` key.